physutils/tasks.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+import pydra
+
+from physutils.io import load_from_bids, load_physio
+from physutils.physio import Physio
+
+LGR = logging.getLogger(__name__)
+LGR.setLevel(logging.DEBUG)
+
+
+@pydra.mark.task
+def transform_to_physio(
+    input_file: str, mode="physio", fs=None, bids_parameters=dict(), bids_channel=None
```

Comment:
It makes a lot of sense to add this as a task, great addition to the phys2denoise workflow and to continue to build up physutils :)

I'd like to suggest changing the name of bids_channel to col_physio_type to increase consistency with the load_from_bids function - since there is not a lot of documentation within the code itself, I think its important to try to maintain intuitive links!

Beyond that, I'm wondering why the task is named transform_to_physio when it can either transform to physio or read-in an existing physio object. Perhaps e.g. generate_physio would be more appropriate? At minimum I think it's helpful to describe the goal of this task and its potential use cases in the description for the PR. Also, please indicate the change type in the PR description :)

Other than these considerations and what Stef mentioned above regarding avoiding a dependency on pydra, this looks good to me!

Reply:
Imo, it's ok as `transform_to_physio`, as even in case it reads a .phys file, it transforms a pickled Physio object file, into a Physio object instance to be used in further tasks.

Also maybe it'd be better to change the variable name in load_from_bids? It's not an interface variable (not used outside the function scope) so I don't think we should take this naming as a basis

Reply:
Kudos on changing name to `generate_physio` or something similar.
Double kudos on adding a good docstring for explanations!